data/mods/Medieval_Stuff/recipes.json

```diff
@@ -0,0 +1,891 @@
+[
+{
+        "type" : "recipe",
+  "result": "scrap_bronze",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_MATERIALS",
+  "skill_used": "fabrication",
+  "difficulty": 2,
+  "time": 5000,
+  "reversible": false,
+  "autolearn": true,
+  "tools": [
+    [
+      [ "goggles_welding", -1 ]
+    ],
+    [
+      [ "crucible", -1 ]
+    ],
+    [
+      [ "welder", 20 ],
+      [ "welder_crude", 30 ],
+      [ "toolset", 30 ],
+      [ "forge", 10],
+      [ "char_forge", 2],
+      [ "oxy_torch", 2]
+    ]
+  ],
+  "components": [
+    [
+      [ "tin", 10 ],
+      [ "solder_wire", 10 ]
+    ],
+    [
+      [ "copper", 50 ],
+      [ "cu_pipe", 1 ]
```

Comment:
"cu_pipe" is 2 volume. Bronze is just 1 volume. You should probably adjust the values, for example by making it work like 1 (volume) tin + 2 (volume) copper = 3 (volume) bronze.